vendor/github.com/containers/common/pkg/auth/cli.go

```diff
@@ -0,0 +1,41 @@
+package auth
+
+import "github.com/spf13/pflag"
+
+// LoginOptions represents the results for flags in login
+type LoginOptions struct {
+	Authfile      string
+	CertDir       string
+	Password      string
+	Username      string
+	StdinPassword bool
+	TlsVerify     bool
+	GetLogin      bool
+}
+
+// LogoutOptions represents the results for flags in logout
+type LogoutOptions struct {
+	Authfile string
+	All      bool
+}
+
+// GetLoginFlags defines and returns login flags for containers tools
+func GetLoginFlags(flags *LoginOptions) *pflag.FlagSet {
+	fs := pflag.FlagSet{}
+	fs.StringVar(&flags.Authfile, "authfile", GetDefaultAuthFile(), "path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override")
+	fs.StringVar(&flags.CertDir, "cert-dir", "", "use certificates at the specified path to access the registry")
+	fs.StringVarP(&flags.Password, "password", "p", "", "Password for registry")
+	fs.StringVarP(&flags.Username, "username", "u", "", "Username for registry")
+	fs.BoolVar(&flags.StdinPassword, "password-stdin", false, "Take the password from stdin")
+	fs.BoolVar(&flags.TlsVerify, "tls-verify", false, "require HTTPS and verify certificates when accessing the registry")
```

Comment:
⚠️ The default **must not** be, in effect, `false`. Even if this did the right thing, there should be tests that this is always interpreted correctly.

It is really possible to have the same `tis-verify` flag defined twice?!

Apparently the “second” definition, whichever that is, is silently ignored by `AddFlagSet`.

That’s just too unreliable for a security-related flag.